Platform/CoffeelakeBoardPkg/Script/StitchLoader.py

```diff
@@ -5,10 +5,15 @@
 # SPDX-License-Identifier: BSD-2-Clause-Patent
 #
 ##
-
+import os
 import sys
 import argparse
-from ctypes import *
+from   ctypes import *
+from   functools import reduce
+
+sys.dont_write_bytecode = True
+sys.path.append (os.path.join('BootloaderCorePkg' , 'Tools'))
+from   IfwiUtility import *
```

Comment:
Looks this patch also did some clean up.
will it still work if copying CoffeelakeBoardPkg out of SBL open?

Reply:
Yes, it will.  This script needs to be executed from the SblOpen folder.